chat-plugins/info.js

```diff
@@ -460,6 +288,13 @@ var commands = exports.commands = {
 		}
 		this.sendReply(buffer);
 	},
+	datahelp:
+		"/data [pokemon/item/move/ability] - Get details on this pokemon/item/move/ability/nature.\n" +
+		"!data [pokemon/item/move/ability] - Show everyone these details. Requires: + % @ & ~",
+	detailshelp:
+		"/details [pokemon] - Get additional details on this pokemon/item/move/ability/nature.\n" +
+		"!details [pokemon] - Show everyone these details. Requires: + % @ & ~",
+	dthelp: function () { this.parse('/help details'); },
```

Comment:
I'm not sure if this is what you meant by recursive function calling, but if so do you want it all on one line like this, or does it look too ugly with all those consecutive tokens?